src/app/core/services/ess-upload-file.service.ts

```diff
@@ -115,10 +118,19 @@ export class EssUploadFileService {
       return true;
     }
   }
-
   addAllSelectedEncs(){
     const maxEncSelectionLimit = this.maxEncSelectionLimit > this.validEncs.length ? this.validEncs.length  : this.maxEncSelectionLimit;
     this.selectedEncs = [...this.validEncs.slice(0,maxEncSelectionLimit)];
   }
-
+  
+  getAvgSizeofENC(encCount:number):string {
```

Comment:
The name of the `getAvgSizeofENC` is misleading. Please change it to `getEstimatedTotalSize`
```suggestion
  getEstimatedTotalSize(encCount:number):string {
```

Reply:
resolved